### ISSUE TITLE
-Threads issues with -OutputFile

### ISSUE THREAD

**Issue by iricigor** [iricigor/GetFolderAge#71]
Two issues:
- the file is at the wrong location if the relative path is provided
- the file contains only last written information (each thread starts new file!)

**Comment by iricigor** [comment]
Test case definition:
```PowerShell
Get-FolderAge . -TestSubfolders -Output File1.csv | Out-Null
Get-FolderAge . -TestSubfolders -Output File2.csv -Threads 2 | Out-Null
(gc File1.csv).Count -eq (gc File2.csv).Count | Should -Be $true
```

**Comment by iricigor** [comment]
Implementation idea:

BEGIN
expand output path
if file exists prepare list of folders to be skipped

PROCESS
folder start - check if it should be skipped
output test - instead of first, check for file existance

**Comment by iricigor** [comment]
resolved with PR #77
